packages/host/src/livegain.js

```diff
@@ -40,7 +40,7 @@ const mountPlugin = (domNode) => {
 	// Create a new instance of the plugin
 	// You can can optionnally give more options such as the initial state of the plugin
 	const instance = await LiveGain.createInstance(audioContext);
-	instance.setState({ enabled: true });
+	window.instance = instance;
```

Comment:
is it still used ? for testing ?

Reply:
It's useful for me to debug the plugin in browser's console...

Reply:
Ok no problem, can you just add a comment over it ?

Reply:
Done